### PR TITLE
E_CLASSROOM-272 [Bugfix] Back button is not a hand cursor

### PR DESCRIPTION
**Pull request by catampongan**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-272

### Definition of Done
- [x] Back button in the subcategory view is not a hand cursor

### Commands to Run
- N/A

#### Main note
- N/A
#### Pages Affected
- http://localhost:3003/categories/:id/sub

### Scenarios/ Test Cases
- [x] Back button in the subcategory view is not a hand cursor
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
image

### CODE REVIEW

**Comment by ErickDoroy** [comment]
### Approve #111 
- [x] In the subcategory view, the back button is a hand cursor.
------
- Total Test Time: 0.25 hr
- Total comments: 0

File: src/pages/student/categories/CategoryDetail/index.module.css [hunk 1]
```diff
@@ -28,24 +28,26 @@
...
+#paginateNum {
   width: 40px;
   height: 34px;
   text-align: center;
   margin-right: 5px;
   border-radius: 2px;
 }
-#BckIcon{
-  margin-right: 10px;
-}
```
Comment (karlolee50) [inline src/pages/student/categories/CategoryDetail/index.module.css:51]: is there still an extra line at the end of the file?

**Review by karlolee50 — COMMENTED** [review]
Review time: 5 minutes
Comment/s: 0

File: src/pages/student/categories/CategoryDetail/index.module.css (hunk 1 above)
Comment (catampongan) [reply src/pages/student/categories/CategoryDetail/index.module.css:51]: yes sir naa

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**